libc-test/build.rs

```diff
@@ -954,6 +955,15 @@ fn test_solarish(target: &str) {
         }
     });
 
+    cfg.skip_type(move |ty| match ty {
+        "sighandler_t"
+        | "lgrp_content_t"
```

Comment:
If these exclusions do end up being required, they should be added up at line 815

Reply:
ah yes there is an existing one.